lib/pengine/unpack.c

```diff
@@ -1108,10 +1112,8 @@ unpack_node_loop(xmlNode * status, bool fence, pe_working_set_t * data_set)
         }
 
         if(process) {
-            crm_trace("Processing lrm resource entries on %shealthy%s node: %s",
-                      fence?"un":"",
-                      (pe__is_guest_or_remote_node(this_node)? " remote" : ""),
-                      this_node->details->uname);
+            crm_trace("Unpacking resource history for %s node %s",
+                      (fence? "unseen " : ""), id);
```

Comment:
There's an extra space after the first %s string.

Reply:
Whoops, thanks